### PR TITLE
Set Cache-Control headers to 1hr

### LINKED ISSUE #5: Cache headers
Thank you for this microservice.

Maybe it will be useful to add `Cache-Control` header to reduce actual request count for users and load for server?

### PR DESCRIPTION
**Pull request by martinklepsch**
Fixes #5 

Set's the `Cache-Control` header to 3600s / 1hr. This should improve performance for icon delivery by allowing Now's CDN to cache responses for up to one hour. 

Potentially the max-age setting could be more aggressive since the data doesn't really change all that much? 3hrs? 12hrs? 3days?

I wasn't able to test this on now.sh yet (getting the error below when running `now`, see [deployment](https://zeit.co/martinklepsch/microicon/fzzdpxnqzu)) but the change is fairly straightforward so maybe that's ok? 😄 

```
> Verifying instantiation in bru1
> [0]
> [0] microicon@1.0.0-b.2 start /home/nowuser/src
> [0] micro -p 3000 server/index.js
> [0]
> [0] micro: Error when importing /home/nowuser/src/server/index.js: Error: Cannot find module '..'
> [0]     at Function.Module._resolveFilename (module.js:547:15)
> [0]     at Function.Module._load (module.js:474:25)
> [0]     at Module.require (module.js:596:17)
> [0]     at require (internal/module.js:11:18)
> [0]     at Object.<anonymous> (/home/nowuser/src/server/index.js:8:19)
> [0]     at Module._compile (module.js:652:30)
> [0]     at Object.Module._extensions..js (module.js:663:10)
> [0]     at Module.load (module.js:565:32)
> [0]     at tryModuleLoad (module.js:505:12)
> [0]     at Function.Module._load (module.js:497:3)
> [0]     at Module.require (module.js:596:17)
> [0]     at require (internal/module.js:11:18)
> [0]     at module.exports.file (/home/nowuser/src/node_modules/micro/lib/module.js:6:11)
> [0]     at module.exports (/home/nowuser/src/node_modules/micro/lib/index.js:9:47)
> [0]     at Object.<anonymous> (/home/nowuser/src/node_modules/micro/bin/micro.js:90:18)
> [0]     at Module._compile (module.js:652:30)
```

### CODE REVIEW

**Comment by martinklepsch** [comment]
Not entirely sure if what I did is correct but it seemed the repo was in a slightly broken state.

- `index.js` wasn't added to the now build causing the missing require pasted above
- The simple-icons repository was moved and no longer has a `gh-pages` branch. I updated the repo and pinned `napa` against a specific commit in order to make the build more predictable.
  - I think simple-icons also changed thew default viewBox from `0 0 16 16` to `0 0 24 24` which I updated in `lib/Icon.js`.

There's a running deployment of this here: https://microicon-yintbnpbdc.now.sh

I think we could still increase the `max-age` to maybe 12 or 24 hours since the assets are really mostly static but not sure if you have any preferences in the regard.

Happy to drop individual commits, rebase, squash or whatever else you'd like.

**Comment by jxnblk** [comment]
Thanks! Yeah, I haven’t touched this in a while so it doesn’t surprise me that simple icons went out of sync; it wasn’t on npm when I put this up. I think the caching could be more aggressive like you suggested and I can also get automatic now deploys set up on another branch if that’s helpful

**Comment by martinklepsch** [comment]
I figured out the deployment issues so the branch deploys aren't necessary (for me). 

> it wasn’t on npm when I put this up

I wasn't aware it's on npm. The changes I made still use `napa` to copy it into `node_modules`. Would you prefer it to use the npm package? 

> I think the caching could be more aggressive like you suggested

Should we set max-age to 24hrs and ship this?

**Comment by jxnblk** [comment]
It looks like it's published now, but haven't tested it out: https://www.npmjs.com/package/simple-icons

**Comment by martinklepsch** [comment]
Yeah I found that package as well... I’ll take a look at removing the `napa` stuff and replacing it with the npm package. 👍

**Comment by martinklepsch** [comment]
@jxnblk Added another commit that removes `napa`, available under https://microicon-cmpilmdzgj.now.sh.

Let me know if anything else needs to be done before this can get merged 👍

File: package.json [hunk 1]
```diff
@@ -34,13 +34,14 @@
     "babel-preset-es2015": "^6.18.0",
     "cheerio": "^0.22.0",
     "material-design-icons": "^3.0.1",
-    "napa": "^2.3.0",
     "nodemon": "^1.11.0",
+    "simple-icons": "^1.9.9",
```
Comment (jxnblk) [inline package.json:38]: This is great, thanks!

**Comment by jxnblk** [comment]
Thanks for this! Going to merge this in and set up automatic deploys on another branch

**Comment by martinklepsch** [comment]
@jxnblk Hey Brent, just wanted to check in if you had a chance to deploy this? I'm still seeing requests being returned without a `Cache-Control` header.

**Comment by martinklepsch** [comment]
Ping 🙂

**Comment by jxnblk** [comment]
I tried to deploy the updates, but something is causing the build to fail on Now. Haven’t had much time to dig in

**Comment by martinklepsch** [comment]
Could you provide more details in what way the build is failing?

**EDIT** I was able to deploy from `master` just fine: https://microicon-cmpilmdzgj.now.sh

**Comment by martinklepsch** [comment]
BTW, I looked into using Now 2.0 but there are currently some limitations with regards to the size of `node_modules` and the bundle in general. `material-design-icons` is unfortunately exceeding the limits with its 356M.

https://spectrum.chat/thread/954196ab-a020-46e8-ad33-f10595b9878d

**Comment by martinklepsch** [comment]
Hey Brent, sorry for bothering you again but have you had a chance to try deploying to Now again or could share some more details on the problems you encountered during deployment?

**Comment by martinklepsch** [comment]
Hey Brent, sorry for pinging again, just wanted to see if there's anything I could do to help get this into production. I don't have much experience with Now but maybe you can also create a team for this project and add me to it so I could look into the deployment issues you described. 

Cheers

**Comment by martinklepsch** [comment]
Bump 🙂

**Pull request merged**